content/documentation/staging/installation-guide/index.adoc

```diff
@@ -298,7 +300,9 @@ deployment:
   - mycorp_.*
 ----
 
-icon:lightbulb[size=1x]{nbsp} If `accessible_namespaces` has an entry with the special value of `+++**+++` (two asterisks), it denotes that Kiali be given access to all namespaces that exist or will be created in the future via a single cluster role. 
+icon:lightbulb[size=1x]{nbsp} If `accessible_namespaces` has an entry with the special value of `+++**+++` (two asterisks), it denotes that Kiali be given access to all namespaces that exist or will be created in the future via a single cluster role. Note that only a single Kiali instance in the cluster should be configured with this special value of `+++**+++`. If multiple Kiali instances are configured with this special value, the Kiali instances may not work correctly.
```

Comment:
suggested change to accompany the one above:

A cluster can have at most one Kiali instance with  `accessible_namespaces` set to the special value of `+++**+++` (two asterisks).    This denotes that Kiali be given access to all namespaces that currently exist, or that will be created in the future, via a single cluster role.   This can be in addition to Kiali instances configured with explicit, mutually exclusive namespaces, as mentioned above.